release/nightly_tests/dataset/gpu_batch_inference.py

```diff
@@ -150,7 +150,4 @@ def __call__(self, batch: Dict[str, np.ndarray]) -> Dict[str, np.ndarray]:
 
 if __name__ == "__main__":
     args = parse_args()
-
-    benchmark = Benchmark("gpu-batch-inference")
-    benchmark.run_fn("batch-inference", main, args)
-    benchmark.write_result()
+    run_benchmark(lambda: main(args), config=vars(args))
```

Comment:
Do you think it would be better to just have the `run_benchmark` function just accept and optional set of args which can be unpacked or otherwise passed into the benchmark function call? Probably not a big deal to just use some lambda functions but think you might be able to get rid of them if it makes sense.